packages/ckeditor5-list/tests/listproperties/ui/listpropertiesview.js

```diff
@@ -567,6 +567,76 @@ describe( 'ListPropertiesView', () => {
 				view.keystrokes.press( keyEvtData );
 				sinon.assert.callCount( keyEvtData.stopPropagation, 4 );
 			} );
+
+			describe( 'special keys keydown event', () => {
```

Comment:
The tests do not accurately assess the true impact of  the `_preventSpecialNumericValues` function. Even if I remove the `event.preventDefault();` logic from this function, all tests still pass.

Reply:
Did you get to my comment about tests? If so, feel free to comment on that as at this point I'm not sure which approach would be better in terms of code quality.
https://github.com/ckeditor/ckeditor5/pull/15851#discussion_r1487968936

Reply:
actually I see now another approach to this, simply extracting only the regexp check, thus making this way simpler in tests. But , given general comment to shift approach, I should rather revert all tests and the logic addition.

Reply:
closing as outdated (removed)